### PR TITLE
Workloads on Topology Page Automation

### PR DESCRIPTION
**Pull request by gajanan-more**
Fix: https://issues.redhat.com/browse/ODC-5469

Problem: Automation to create different types of Workloads

Solution: This script will create different types of workloads and check whether they are visible on Topology page or not

Test Setup:
1. Add @topology under env.tags in cypress.json
2. Run `yarn run test-cypress-devconsole` in `frontend`
3. Execute feature file: **frontend/packages/dev-console/integration-tests/features/topology/workloads-on-topology.feature**

Test Execution Screenshot:
Screenshot from 2021-02-09 09-44-15

### CODE REVIEW

File: frontend/packages/dev-console/integration-tests/features/topology/workloads-on-topology.feature [hunk 1]
```diff
@@ -0,0 +1,29 @@
+Feature: Topology
+    User will be able to create different types of workloads like Cron Job, Job and Pod and will be able to see it on Topology page
+
+
+    Background:
+        Given user is at developer perspective
+        And user is at the Topology page
+        
+
+    @topology
```
Comment (makambalaji) [inline frontend/packages/dev-console/integration-tests/features/topology/workloads-on-topology.feature:10]: @topology tag is applicable for all scenarios, so I recommend you to add on feature level [suggested above]. It throws error if you try to run `yarn run gherkin-lint` command for the same

File: frontend/packages/dev-console/integration-tests/features/topology/workloads-on-topology.feature [hunk 2]
```diff
@@ -0,0 +1,29 @@
+Feature: Topology
```
Comment (makambalaji) [inline frontend/packages/dev-console/integration-tests/features/topology/workloads-on-topology.feature:1]:
```suggestion
@topology
Feature: Topology
```

File: frontend/packages/dev-console/integration-tests/features/topology/workloads-on-topology.feature [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+    @topology
+    Scenario: Create Cron Job type workload
+        When user has created or selected namespace "aut-workloads-admin" 
+        And user applies cronjob YAML
+        And user is at namespace "aut-workloads-admin"
+        Then user will see cron job with name "example-cronjob" on topology page
+
+
+    @topology
```
Comment (makambalaji) [inline frontend/packages/dev-console/integration-tests/features/topology/workloads-on-topology.feature:18]: Remove the tag

File: frontend/packages/dev-console/integration-tests/features/topology/workloads-on-topology.feature [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    Scenario: Create Job type workload
+        When user applies job YAML
+        And user is at namespace "aut-workloads-admin"
+        Then user will see job with name "example-job" on topology page
+
+
+    @topology
```
Comment (makambalaji) [inline frontend/packages/dev-console/integration-tests/features/topology/workloads-on-topology.feature:25]: Remove the tag

File: frontend/packages/dev-console/integration-tests/testData/YAMLs/create-cronjob.yaml [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+      template:
+        spec:
+          containers:
+            - name: hello
+              image: busybox
+              args:
+                - /bin/sh
+                - '-c'
+                - date; echo Hello from the Kubernetes cluster
+          restartPolicy: OnFailure
```
Comment (makambalaji) [inline frontend/packages/dev-console/integration-tests/testData/YAMLs/create-cronjob.yaml:19]: Prettier issue, please fix this

File: frontend/packages/dev-console/integration-tests/testData/YAMLs/create-job.yaml [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+    spec:
+      containers:
+        - name: pi
+          image: perl
+          command:
+            - perl
+            - '-Mbignum=bpi'
+            - '-wle'
+            - print bpi(2000)
+      restartPolicy: Never
```
Comment (makambalaji) [inline frontend/packages/dev-console/integration-tests/testData/YAMLs/create-job.yaml:20]: Empty line is required at the end of every file, please make sure of it

**Review by makambalaji — COMMENTED** [review]
Please update the description with the file name to select on cypress dashboard

File: frontend/packages/dev-console/integration-tests/features/topology/workloads-on-topology.feature (hunk 1 above)
Comment (gajanan-more) [reply frontend/packages/dev-console/integration-tests/features/topology/workloads-on-topology.feature]: Updated

File: frontend/packages/dev-console/integration-tests/features/topology/workloads-on-topology.feature (hunk 2 above)
Comment (gajanan-more) [reply frontend/packages/dev-console/integration-tests/features/topology/workloads-on-topology.feature:2]: Updated

File: frontend/packages/dev-console/integration-tests/features/topology/workloads-on-topology.feature (hunk 3 above)
Comment (gajanan-more) [reply frontend/packages/dev-console/integration-tests/features/topology/workloads-on-topology.feature]: Updated

File: frontend/packages/dev-console/integration-tests/features/topology/workloads-on-topology.feature (hunk 4 above)
Comment (gajanan-more) [reply frontend/packages/dev-console/integration-tests/features/topology/workloads-on-topology.feature]: Updated

File: frontend/packages/dev-console/integration-tests/testData/YAMLs/create-cronjob.yaml (hunk 5 above)
Comment (gajanan-more) [reply frontend/packages/dev-console/integration-tests/testData/YAMLs/create-cronjob.yaml]: Updated

File: frontend/packages/dev-console/integration-tests/testData/YAMLs/create-job.yaml (hunk 6 above)
Comment (gajanan-more) [reply frontend/packages/dev-console/integration-tests/testData/YAMLs/create-job.yaml:20]: Updated

**Pull request merged**